core/src/main/java/com/linecorp/armeria/common/CookieUtil.java

```diff
@@ -251,6 +251,18 @@ static CookieBuilder initCookie(Logger logger, boolean strict,
     }
 
     // The methods newly added in the fork.
+    static Cookies fromCookieHeaders(ImmutableSet.Builder<Cookie> builder,
```

Comment:
Let's remove this method and put this logic in
`fromCookieHeaders(boolean strict, Iterable<String> cookieHeaders) {...}`